.github/workflows/build.yml

```diff
@@ -86,13 +89,15 @@ jobs:
 
           failed_commits=""
 
+          source finch-flight-software-env.sh
+
           for commit in $commits; do
             echo "::group::Building $commit"
 
             git checkout $commit
             rm -rf build/
             set +e
-            west -v build -b ${{ matrix.board }} ${{ matrix.path }} -p $EXTRA_ARGS
+            west -v build -b ${{ matrix.board }} ${{ matrix.path }} -p $EXTRA_ARGS -DBOARD_ROOT=$FINCH_FLIGHT_SOFTWARE_ROOT
```

Comment:
It’s probably better to add `-DBOARD_ROOT=$FINCH_FLIGHT_SOFTWARE_ROOT` to $EXTRA_ARGS.